ui/plugins/controller.js

```diff
@@ -141,6 +152,59 @@ treeherder.controller('PluginCtrl', [
                           }
                           return result;
                         }, []);
+                        if ($scope.job_ids !== null) {
+                            var signatureList = _.keys($scope.job_ids);
+                            var summarySignature = [];
+                            $q.all(_.chunk(signatureList, 20).map(function(signatureChunk) {
+                                var url = '/performance/signatures/?signature=' + signatureChunk.pop();
+                                signatureChunk.forEach(function(signature) {
+                                    url = url + '&signature=' + signature;
+                                });
+                                return $http.get(thUrl.getProjectUrl(url, $scope.repoName)).then(
+                                    function(response) {
+                                        // In general, we only need to show the summary signature. But
+                                        // some other test like 'main_rss' of tp5o been required also.
+                                        // For now, I only know 'Private Bytes', 'XRes' and 'Main_RSS'
```

Comment:
We should probably do the same thing we do for compare view here: get a giant list of all the signatures, and then filter out those subtests which have a summary associated with them:

https://github.com/mozilla/treeherder/blob/master/ui/js/perf.js#L171

Any hardcoded list like this is bound to be incomplete and out of date.